### PR TITLE
updated Creating a release from a build server plug-in

### PR DESCRIPTION
**Pull request by FinnianDempsey**
Changed to clarify intention behind each field. Just added a little extra info about each

@harrisonmeister - Just a suggestion, feel free to adjust to how you like! 

https://octopus.zendesk.com/agent/tickets/86312

### CODE REVIEW

**Comment by harrisonmeister** [comment]
@FinnianDempsey - I made some changes. You made specific references to ADO in a generic section for all build server plugins, so I have removed any ADO-specific content and tried to generalize the points you were making.

Take a look at my changes and see what you think

**Comment by FinnianDempsey** [comment]
Good point!! Love the changes!

**Review by harrisonmeister — APPROVED** [review]

File: docs/projects/version-control/creating-and-deploying-releases-version-controlled-project.md [hunk 1]
```diff
@@ -18,12 +18,17 @@ When you create a release via the UI, you must specify a branch name. Octopus wi
...
+The use of these fields can change depending on where the version-controlled project's OCL files are stored:
+
+1. If the OCL files are stored in the **same repository** as the application(s) being built, it's likely that a specific commit that relates to any artifacts created as part of the build itself should be used when creating the release. In this scenario, you should provide both the Git Reference and Git Commit hash of the executing build. This ensures that the release will use the correct version of the project, and won't include any potential changes made to the `HEAD` of the branch *before* the build has completed.
+2. If the OCL files are stored in a **different repository** than the application(s) being built, a specific branch or tag can identify which version of the project to use when creating the release. In this case, you would provide the Git Reference where the OCL files are stored, and not the Git Commit hash. e.g. Use the `main` branch, regardless of the location of the repository where the application(s) are being built as they are different.
+
+It is highly recommended that you provide the commit and not just the branch in both cases.
```
Comment (dylanlerch) [inline docs/projects/version-control/creating-and-deploying-releases-version-controlled-project.md:31]: Point 2 nicely covers how and why you'd point to just a branch, and then we are saying users still probably shouldn't. Perhaps we can just drop this line all together?

Comment (FinnianDempsey) [reply docs/projects/version-control/creating-and-deploying-releases-version-controlled-project.md]: I agree, change made if @harrisonmeister also agrees!

Comment (harrisonmeister) [reply docs/projects/version-control/creating-and-deploying-releases-version-controlled-project.md]: Works for me

**Pull request merged**